pom.xml

```diff
@@ -2239,7 +2239,7 @@
                 <delta.version>2.4.0</delta.version>
                 <spark.version>3.4.1</spark.version>
                 <spark.binary.version>3.4</spark.binary.version>
-                <maven.plugin.scalatest.exclude.tags>org.scalatest.tags.Slow,org.apache.kyuubi.tags.IcebergTest</maven.plugin.scalatest.exclude.tags>
```

Comment:
is there any context to remove this tag ?

Reply:
> is there any context to remove this tag ?

The test cases containing the `IcebergTest` tag have executed successfully, I think we can remove it.